pkg/cmd/experimental/syncgroups/groupdetector/groupdetector.go

```diff
@@ -76,13 +80,13 @@ type CompoundDetector struct {
 }
 
 func (l *CompoundDetector) Exists(ldapGrouUID string) (bool, error) {
-	conclusion := false
+	conclusion := true
```

Comment:
Add a test that makes sure that no locators results in a `false` to make sure this never slips.